### DISCUSSION TITLE
What's the reason to pin powershell core to version 7.2(.7) instead of using 7.3?

### DISCUSSION THREAD

**Discussion by HUMBERP** [actions/virtual-environments#6617] [Installed software]
Currently I use the runner-images repo to add my own customization to build a self-hosted runner image and have updated my environment to use powershell core 7.3.

For a short period of time 7.3 was installed, but now it's reverted in a way that I can't optionally opt-it to 7.3 :-(

**Comment by mikhailkoliada** [comment]
Hello!

Lots of modules have not been ported to 7.3 yet (like Az or azure-cli as a whole had problems interacting with 7.3).

Plus, there were changed like [this](https://learn.microsoft.com/en-us/powershell/scripting/learn/experimental-features?view=powershell-7.3#psnativecommandargumentpassing) breaking lots of people.

**Reply by HUMBERP** [reply]
Thanks for the quick response.
